src/Elcodi/Common/CommonBundle/Command/ElcodiInstallCommand.php

```diff
@@ -0,0 +1,245 @@
+<?php
+
+/*
+ * This file is part of the Elcodi package.
+ *
+ * Copyright (c) 2014-2015 Elcodi.com
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Feel free to edit as you please, and have fun.
+ *
+ * @author Marc Morera <yuhu@mmoreram.com>
+ * @author Aldo Chiecchia <zimage@tiscali.it>
+ * @author Elcodi Team <tech@elcodi.com>
+ */
+
+namespace Elcodi\Common\CommonBundle\Command;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\StringInput;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+/**
+ * Class ElcodiInstallCommand
+ */
+class ElcodiInstallCommand extends Command
+{
+    /**
+     * @var KernelInterface
+     *
+     * Kernel
+     */
+    private $kernel;
+
+    /**
+     * @var ObjectManager
+     *
+     * Location entity manager
+     */
+    private $locationEntityManager;
+
+    /**
+     * Construct
+     *
+     * @param KernelInterface $kernel                Kernel
+     * @param ObjectManager   $locationEntityManager Location entity manager
+     */
+    public function __construct(
+        KernelInterface $kernel,
+        ObjectManager $locationEntityManager
+    ) {
+        parent::__construct();
+
+        $this->kernel = $kernel;
+        $this->locationEntityManager = $locationEntityManager;
+    }
+
+    /**
+     * Configures the current command.
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('elcodi:install')
+            ->setDescription('Installs Elcodi in your dev environment')
```

Comment:
dev or testing ? the description is not consistent with the environment set up for the command.